### PR TITLE
Fix issue #11: wrong filename path

### LINKED ISSUE #11: Unhandled rejection TypeError: Path must be a string. Received undefined
# Issue Title
Bug caused due to 0.1.7 from https://github.com/themgoncalves/react-loadable-ssr-addon/commit/6c18c7ae07074cc25a2889e5c9a4b97acab807d1

## Expected Behavior
No error

## Current Behavior
```bash
Unhandled rejection TypeError: Path must be a string. Received undefined
    at assertPath (path.js:28:11)
    at Object.resolve (path.js:1168:7)
    at ReactLoadableSSRAddon.get (/home/travis/build/endiliey/endiliey.github.io/website/node_modules/react-loadable-ssr-addon/lib/ReactLoadableSSRAddon.js:246:31)
    at ReactLoadableSSRAddon.writeAssetsFile (/home/travis/build/endiliey/endiliey.github.io/website/node_modules/react-loadable-ssr-addon/lib/ReactLoadableSSRAddon.js:189:25)
```

## Possible Solution

I think filename should not be defined here.
https://github.com/themgoncalves/react-loadable-ssr-addon/blob/6c18c7ae07074cc25a2889e5c9a4b97acab807d1/source/ReactLoadableSSRAddon.js#L67

It should be 
```js
const filename = this.options.filename
```

This is the line that causes the error
https://github.com/themgoncalves/react-loadable-ssr-addon/blob/6c18c7ae07074cc25a2889e5c9a4b97acab807d1/source/ReactLoadableSSRAddon.js#L80

Previously it was
https://github.com/themgoncalves/react-loadable-ssr-addon/blob/7b8721db4da23fed0657e5b1755014b7f5daaf62/source/ReactLoadableSSRAddon.js#L142

I can submit a PR

### PR DESCRIPTION
**Pull request by endiliey**
## Description
Fix #11 

This is the line that causes the error. v0.1.7
https://github.com/themgoncalves/react-loadable-ssr-addon/blob/6c18c7ae07074cc25a2889e5c9a4b97acab807d1/source/ReactLoadableSSRAddon.js#L67
https://github.com/themgoncalves/react-loadable-ssr-addon/blob/6c18c7ae07074cc25a2889e5c9a4b97acab807d1/source/ReactLoadableSSRAddon.js#L80

Previously it was like this in v0.1.6
https://github.com/themgoncalves/react-loadable-ssr-addon/blob/7b8721db4da23fed0657e5b1755014b7f5daaf62/source/ReactLoadableSSRAddon.js#L142

## Why
See #11

### CODE REVIEW

File: source/ReactLoadableSSRAddon.js [hunk 1]
```diff
@@ -60,11 +60,12 @@ export default class ReactLoadableSSRAddon {
    * @returns {string} - Output path containing path + filename.
    */
   get manifestOutputPath() {
-    if (path.isAbsolute(this.options.filename)) {
-      return this.options.filename;
+    const filename = this.options.filename;
```
Comment (endiliey) [inline source/ReactLoadableSSRAddon.js]: Alternatively, 

```js
const { filename } = this.options;
```

Comment (themgoncalves) [reply source/ReactLoadableSSRAddon.js]: Looks good your fix, but could you please switch the implementation to the `destructuring assignment` one?
Just for the sake of code consistence :)

**Comment by themgoncalves** [comment]
Thanks for spotting and fix this issue caused by release of `v0.1.7` 💪

Just got one comment due to the `code consistence`, once you fix I will merge it and release a new version with this `hotfix`.

**Comment by endiliey** [comment]
@themgoncalves done. you might want to deprecate 0.1.7 since it will break

**Review by themgoncalves — APPROVED** [review]
LGTM

**Pull request merged**